### DISCUSSION TITLE
请问hyperV运行paopaogw连接以后是怎样的呢？

### DISCUSSION THREAD

**Discussion by lesnolie** [kkkgo/PaoPaoGateWay#69] [问和答]
您好，检查了一下午，发现配置都没有问题，但是paopaogw连接以后的截图是：
![image](https://github.com/kkkgo/PaoPaoGateWay/assets/81410185/376f87d8-08d7-4f07-af48-c722ab0335db)
请问这正常吗？

**Comment by kkkgo** [comment]
新建虚拟机的时候选择第一代，关闭安全引导。

**Reply by lesnolie** [reply]
非常感谢，管用了！
我是设置了第二代。
